gradio/event_queue.py

```diff
@@ -120,19 +122,13 @@ def push(cls, event: Event) -> int | None:
         cls.EVENT_QUEUE.append(event)
         return len(cls.EVENT_QUEUE) - 1
 
-    @classmethod
-    def clean_job(cls, event: Event):
-        cls.ACTIVE_JOBS[cls.ACTIVE_JOBS.index(event)] = None
-
     @classmethod
     async def clean_event(cls, event: Event) -> None:
-        # Using mutex to avoid editing a list in use
-        async with cls.DELETE_LOCK:
-            # TODO: Might log number of cleaned events in the future
-            try:  # TODO: Might search with event hash to speed up
+        if event in cls.EVENT_QUEUE:
+            async with cls.DELETE_LOCK:
                 cls.EVENT_QUEUE.remove(event)
-            except ValueError as er:
-                print(f"{er}, {event}")
+        elif event in cls.ACTIVE_JOBS:
+            cls.ACTIVE_JOBS[cls.ACTIVE_JOBS.index(event)] = None
```

Comment:
Should we still have an `try`/`else` clause for other errors that might happen if removing the event? Or was the only source of errors?

Reply:
I don't see why there should be a try/catch on something like this

Reply:
Don't see how any other error could happen